schemas/ws/ws_genome_features.yaml

```diff
@@ -0,0 +1,34 @@
+name: ws_genome_features
+type: vertex
+schema:
+  "$schema": "http://json-schema.org/draft-07/schema#"
+  type: object
+  required:
+    - _key
+    - feature_id
+    - workspace_id
+    - object_id
+    - version
+  properties:
+    _key:
+      type: string
+      description: The UPA and feature ID for this data
+      examples: ["35414:73:1_RSP_4039"]
+      pattern: "^\\d+:\\d+:\\d+_\\w*$"  # may need to expand the feature ID part
+    feature_id:
+      type: string
+      description: The unique ID of the feature within the genome
+      examples: ["RSP_4039"]
+      pattern: "^\\w*$" # may need to expand
```

Comment:
I'd lean towards not doing a regex here. I'm not sure how it'd benefit us and would probably slow down the schema validation.

Reply:
Per discussion, decided to keep this. It prevents weird characters (like space) winding up in the ID, and there's already a more complex regex on _key, so this doesn't add much more overhead. Neither of us have strong opinions though.